src/components/Routes/Find/Search/Search.js

```diff
@@ -171,26 +171,26 @@ class Search extends Component {
       <>
         <h1 className="accessible">Search</h1>
         <form onSubmit={(event) => this.submitSearchForm(event, userIp, userAgent, limit)} className="form">
-          {formElementsArray.map((formElement) => {
+          {formElementsArray.map(({id, config}) => {
             return (
               <FormElement
-                key={formElement.id}
-                id={formElement.id}
-                widths={formElement.config.widths}
-                elementType={formElement.config.elementType}
-                elementConfig={formElement.config.elementConfig}
-                hasGeolocateButton={formElement.config.hasGeolocateButton}
-                label={formElement.config.label}
-                value={formElement.config.value}
+                key={id}
+                id={id}
+                widths={config.widths}
```

Comment:
Seeing as all these values you're referencing on `config` match their prop names (e.g. `widths={config.widths}`), you can replace them all with one object spread expression

```suggestion
                {...config}
```

If there are only some properties you'd like to pass through, it's a great moment for a utility library like lodash:

```
{..._.pick(config, ['widths', 'elementType', 'elementConfig', 'hasGeolocateButton', 'label', 'value'])}
```

or even more clearly, you could define the keys you're wanting to `pick` away from the usage:

```
const formElementsArray = forms.createFormElementsArray(this.state.form);
const formElementConfigPropsToPass = ['widths', 'elementType', 'elementConfig', 'hasGeolocateButton', 'label', 'value'];

return (
...
<FormElement
  key={id}
  id={id}
  {..._.pick(config, formElementConfigPropsToPass)}
...
```

Reply:
@shoelessness Ahh ok, yeah that definitely cleans it up quite a bit. I'll do that in all the places I can find.

Reply:
@shoelessness oh, so would you suggest importing the entire lodash, or just the ones I want like this?

```import _isEqual from 'lodash/isEqual';
import _pick from 'lodash/pick';```

Reply:
Defo import only what you need unless there’s tree shaking set up in your webpack config.
Tree shaking is a bit complicated but once it’s done, it helps bundle size stay small

Reply:
Ok, that's what I figured, but just wanted to check. Thanks!